scripts/publish-linux-amd64

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
+
+source $(dirname $0)/version
+
+docker image push rancher/fluentd:${DRONE_TAG}-linux-amd64
```

Comment:
We do not need this file 
```sh
docker image push rancher/fluentd:${DRONE_TAG}-linux-amd64
```
This can be added above

Reply:
deleted the file and folded the push command into the makefile